db/migrate/20220919163350_create_arrests.rb

```diff
@@ -0,0 +1,16 @@
+class CreateArrests < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tulsa_blotter_arrests do |t|
+      t.references :tulsa_blotter_inmates, null: true, foreign_key: true
+      t.string :arrest_date
```

Comment:
Do we want to store these date fields as strings or parse them into dates? Not sure what's best